src/products/components/ProductList/ProductList.tsx

```diff
@@ -130,6 +138,46 @@ export const ProductList: React.FC<ProductListProps> = props => {
   );
   const numberOfColumns = 2 + settings.columns.length;
 
+  const getProductPrice = (
+    variants: ProductList_products_edges_node_variants[]
+  ) => {
+    if (!variants.length) {
+      return null;
+    }
+
+    const { max, min } = getProductPriceRange(variants);
```

Comment:
But product will still have pricerange field at product AFAIK. Why not use it?

Reply:
![image](https://user-images.githubusercontent.com/5188636/84891950-03e64c80-b09d-11ea-8661-cbe97ec1e5ef.png)

Reply:
I didn't know about it. I'll change it

Reply:
fixed, changed to priceRangeUndiscounted and gross price is displayed